gnosis/safe-react
# Sdk modules management

## What it solves
Resolves https://github.com/gnosis/safe-react/issues/2797

## How this PR fixes it
Uses the safe-core-sdk to call the methods `getEnableModuleTx` and `getDisableModuleTx`

## How to test it
- Check that the modules list is empty in Settings > Advanced
- Create a new Spending Limit for the first time in Settings > Spending Limit
- Check that a new Safe module was added to the list in Settings > Advanced
- Remove that module using the actions in the right side of the modules table
